### DISCUSSION TITLE
IRQ pin state at startup

### DISCUSSION THREAD

**Discussion by okhsunrog** [Ralim/usb-pd#19] [General]
So I'm going to use this lib with eps32-c3. And after I was out of free pins, I noticed, that I need an additional pin IRQ for FUSB302MPX.
I'm going to use GPIO9, which is a strapping pin and is used to get into download mode of the chip. I need it be pulled up to be high at startup, but this is okay, because FUSB302MPX also wants IRQ pin, as well as I2C to be pulled up, according to the datasheet. Now the question is: will it interfere with the esp32-c3 startup? if IRQ is low at startup - that's some bad news for me. maybe I'm going to delay the startup of FUSB302MPX using a simple RC-curcuit in that case. 
If you have some free time - could you measure the state of IRQ pin at the startup? Is it idle of pulled down?

**Comment by Ralim** [comment]
Off memory it was high impedance at startup.
The FUSB takes a little bit of time to init before it can do anything so you should be fine anyway.
You could alternatively put SDA or SCL on GPIO9 fwiw.

**Reply by okhsunrog** [reply]
Thanks! I'd better connect SDA or SCL to GPIO9 then
